### ISSUE TITLE
Mutliple values properties as filter

### ISSUE THREAD

**Issue by davidmpaz** [typicode/json-server#843]
Hi,

thanks for this wonderfull project!! On docs I can see under **Filter**: `GET /posts?id=1&id=2`, I wonder myself wheter this other syntax could be achieved: `GET /posts?id=1,2`

thanks in advance,
David

**Comment by lipengzhou** [comment]
@davidmpaz dsadsa

**Comment by SumitGoel-ITT** [comment]
Did you find any solution on this? 
I am looking for something similar

**Comment by davidmpaz** [comment]
No. Maybe a similar approach like this [comment](https://github.com/typicode/json-server/issues/844#issuecomment-415850322) could help though.

Dont know now, maybe some kind of middleware to preprocess request parameter and transform to the valid one can help also
